metis-dereference/metis-dereference-common/src/main/java/eu/europeana/metis/dereference/DereferenceUtils.java

```diff
@@ -26,17 +38,8 @@
 import eu.europeana.corelib.definitions.jibx.SameAs;
 import eu.europeana.corelib.definitions.jibx.TimeSpanType;
 import eu.europeana.corelib.definitions.jibx.WebResourceType;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Set;
-import org.apache.commons.io.IOUtils;
-import org.apache.commons.lang3.StringUtils;
-import org.jibx.runtime.BindingDirectory;
-import org.jibx.runtime.IBindingFactory;
-import org.jibx.runtime.IUnmarshallingContext;
-import org.jibx.runtime.JiBXException;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
+import eu.europeana.enrichment.api.external.model.EnrichmentBase;
+import eu.europeana.enrichment.utils.EntityMergeUtils;
 
```

Comment:
In this class there is a System.exit(-1). That may have unexpected consequences if other applications start using it... maybe we should change this?

Reply:
Again, it's in the static block, so I don't really mind this.

Reply:
I fixed it.